README.md

```diff
@@ -39,10 +39,30 @@
 * SXSW
 * Bear claws
 * Pearl Jam
+* Freshcraft bar
 * HipChat
 * Trend IMSS
 * Joe Nacchio
 * Ubuntu
 * Complainy Hulu customers
 * Places called "Fish Tacos"
 * Going south of Alameda
+
+## Quotes
+* Wait a tic
+* Al, I'll miss you the most
+* I have the sys admin physique
+* Everything's coming up Milhouse
+* It's not "slash user", it's "slash U, S, R"
+* Cold food is for peasants
```

Comment:
Cold sandwiches are for peasants. Words do indeed have meaning.

Reply:
I distinctly remember:
>Cold food is for peasants. We invented fire for a reason.

While you may agree about words; there's a reason y'all called me "Words-have-meaning-Matt". :expressionless: Just sayin'.

Reply:
For what it's worth, human memory is fallible, even yours my good Sir. This statement misrepresents my actual feelings on cold food. I have made the standard line on fire countless times, in regards to sandwiches. Words do indeed have meaning, food includes many items which have no business being heated, at least in their final form:
 - Beer 
 - Ice cream
 - Chocolate milk

Reply:
`Cold sandwiches are for peasants` doesn't quite have the same ring to it so I think I'll just remove the line. FWIW, my memory is in alignment with @elementalvoid's on this one.

Reply:
I feel the need to sit in a triangle and vehemently agree ; )

I'll split the difference: while I recall the quote differently than @rpkish I _do agree_ it was always made _in reference to sandwiches_ (as that was always the conversation topic).